allennlp/data/dataset_readers/dataset_utils/ontonotes.py

```diff
@@ -429,3 +430,81 @@ def _process_span_annotations_for_word(annotations: List[str],
             # Exiting a span, so we reset the current span label for this annotation.
             if ")" in annotation:
                 current_span_labels[annotation_index] = None
+
+
+def bio_tags_to_spans(tag_sequence: List[str],
```

Comment:
Will we want this outside of ontonotes?  I'm just wondering if this is the best place for this code to live.

Reply:
We can move it out once it starts to be used elsewhere?

Reply:
Sure, seems reasonable.